recipes/data-prep-kit/end-2-end-code-data-prep-finetuning.ipynb

```diff
@@ -0,0 +1,869 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "NbF_Zw3KBazf"
+   },
+   "source": [
+    "# **Demo on building data prep pipeline for model fine tuning** "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"https://colab.research.google.com/github/IBM/data-prep-kit/blob/dev/examples/notebooks/code/sample-notebook.ipynb\">\n",
```

Comment:
This cell will be pointing to the wrong place when merged in this repo, and colab buttons in the notebook is not in line with our style guide. Please drop this cell

Reply:
File path is fixed. We may not be able to drop this cell. Can you pls suggest on how you would like the colab button to be?

Reply:
Please drop the cell. Just delete it.